doc/source/ray-core/scheduling/resources.rst

```diff
@@ -0,0 +1,154 @@
+.. _core-resources:
+
+Resources
+=========
+
+Ray allows you to seamlessly scale your applications from a laptop to a cluster without code change.
+It is possible because of Ray resources.
+Ray resources abstract away physical machines and let you express your computation in terms of resources instead of machines,
+while the system manages scheduling and autoscaling based on resource requests.
+
+A resource in Ray is a key-value pair where the key denotes a resource name, and the value is a float quantity.
+For convenience, Ray has native support for CPU, GPU, and memory resource types; CPU, GPU and memory are called **pre-defined resources**.
+Besides those, Ray also supports :ref:`custom resources <custom-resources>`.
+
+Ray resources are **logical** and don’t need to have 1-to-1 mapping with physical resources.
```

Comment:
Maybe we could start with a new sub-section like this to explain the differences more clearly?
### Logical resources and physical resources
- Logical resources: ...
- Physical resources:...

Ray resources are **logical** and don’t need to have 1-to-1 mapping with physical resources....

Reply:
I gave it a try, lmk if it looks better.